php/libraries/VisitTest.php

```diff
@@ -0,0 +1,380 @@
+<?php
+/**
+ * This file contains the unit test for Visit_Type class.
+ *
+ * PHP Version 7
+ *
+ * @category Main
+ * @package  Main
+ * @author   Mélanie Legault <melanie.legault2@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+require_once __DIR__ . "/NDB_Factory.class.inc";
+require_once __DIR__ . "/../../test/bootstrap.php";
+require_once __DIR__ . "/Visit.class.inc";
+require_once __DIR__ . "/Database.class.inc";
+require_once __DIR__ . "/testConfig.inc";
+
+// TODO
+// - creer 4 visit_type et garder les IDs
+// - récupérer un visit_type par son ID
+// - ajouter des projects subproject (p, x), (x, sp), (p, sp)
+// - récupérer la liste des visit type
+// - récupérer la liste des visit type pour 1 project (p, x), (x, sp), (p,s sp)
+// - modifier un visit type label (avec et sans legacy label)
+// - supprimer les visit_type_project_subproject 1 par 1 et vérifier la liste
+//   entre chaque
+
+/**
+ * This class test the Visit class
+ *
+ * @category Main
+ * @package  Main
+ * @author   Mélanie Legault <melanie.legault2@mcgill.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+class VisitTest
```

Comment:
Should be a PHPUnit test so that it gets run by Travis with the rest of the test suite